bananas_api/helpers/api_schema.py

```diff
@@ -64,6 +64,18 @@ def __call__(self, value):
         return super().__call__(value)
 
 
+class ValidateBytesLength(validate.Length):
+    def __call__(self, value):
+        super().__call__(value.encode())
+
+    def _format_error(self, value, message):
+        # There were no issues encoding it in __call__,
```

Comment:
personally, I would just do: `value = value.decode()`, and call the parent.

Given you did `encode()` without parameter, no need to do `decode()` with parameter.